unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs

```diff
@@ -56,9 +56,11 @@ public class ExploreV2MenuComponentController : IExploreV2MenuComponentControlle
     internal BaseVariable<bool> emotesVisible => DataStore.i.HUDs.emotesVisible;
     internal BaseVariable<bool> chatInputVisible => DataStore.i.HUDs.chatInputVisible;
     internal BooleanVariable playerInfoCardVisible => CommonScriptableObjects.playerInfoCardVisibleState;
+    internal MouseCatcher mouseCatcher;
 
     public void Initialize()
     {
+        mouseCatcher = SceneReferences.i?.mouseCatcher;
```

Comment:
Shouldn't SceneReferences.i always exist?

Reply:
Yes, in runtime... but in test mode (where I don't want to test the mouse catcher) may not exist. So we add this "security" check.